Lanchonete/Lanchonete/ Features/Confirmation/Presentation/ConfirmationViewController.swift

```diff
@@ -39,12 +37,27 @@ final class ConfirmationViewController: UIViewController {
     }
     
     @IBAction func personalizeButtonTapped(_ sender: Any) {
+        self.performSegue(withIdentifier: Segues.confirmationToCustomization, sender: presenter?.getSandwichIngredients())
```

Comment:
We usually tell the `presenter` that the button was tapped and the presenter decides to move to another screen. And the navigation is handled by a router class. I don't see any router class, I guess you can have the navigation in the view controller for simplification, but maybe it could go through the `presenter` and the presenter tells the view controller that it should navigate to said screen with said options.

Reply:
Hey Carlos, good tip, I think implementing like your suggestion would be kinda tricky since I never did like that. Do you want me to implement it now or can we leave this suggestion for later?